### PR TITLE
Fixing a parsing error.

### PR DESCRIPTION
**Pull request by serg-bloim**
The error happens when packet contains field 'id' that is not in CHANNEL_NAMES

### CODE REVIEW

File: SpektrumPacketAnalyzer.py [hunk 1]
```diff
@@ -129,16 +129,24 @@ def parse_channels(self, proto):
         frames2 = [frames[i:i + 2] for i in range(0, len(frames), 2)] # split 'frames' array into size-two chunks 
         return [self.parse_channel(frame2, proto) for frame2 in frames2]
 
+
+    def get_channel_name(self, id:int):
+        if 0 <= id < len(CHANNEL_NAMES):
+            return CHANNEL_NAMES[id]
+        else:
+            return "NOT_IDENTIFIED"
```
Comment (nbes4) [inline SpektrumPacketAnalyzer.py]: Could you change this string from `"NOT_IDENTIFIED"` to `"Unknown"`? I think that would fit better with the rest 👍

**Review by nbes4 — CHANGES_REQUESTED** [review]
Hi @serg-bloim! Thank you so much for your PR! I reviewed it and it looks good, only a very-small nitpick regarding the returned string. 😄

File: SpektrumPacketAnalyzer.py (hunk 1 above)
Comment (serg-bloim) [reply SpektrumPacketAnalyzer.py]: Sure thing, let me change it.

**Review by nbes4 — APPROVED** [review]

**Comment by nbes4** [comment]
Awesome, I squashed the commits and merged them. Thanks again, @serg-bloim. 🎉

**Pull request merged**